jquery.overlay.js

```diff
@@ -242,9 +264,12 @@
         }
       },
 
-      register: function (strategies) {
+      register: function (strategies, allowOverlapping) {
```

Comment:
I'd rather pass in an `options` object here, rather than a single boolean. If we customize this further there could be more customizations and maintaining a method with lots of optional arguments can be a real pain.